Aptoide/aptoide-client-v8
# [ASV-1188] After installing Aptoide, the beta preferences are now set…

**What does this PR do?**

  Sets beta version to false when the user updates Aptoide from a version below 9.4.0.0

**Database changed?**

   Yes, Realm Schema version was upgraded to 8096

**Where should the reviewer start?**

RealmToRealmMigration

**How should this be manually tested?**

Unistall any aptoide version you have, in gradle.properties change realm schema version to 8095 and install aptoide.
Then change the value of beta versions to true in settings
Then in gradle.properties change it to 8096
Run again and check if the value was set to false

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1188

**Code Review Checklist**
